examples/templates/configuration/raw-keyrings/raw-rsa-keyring-from-encoded-key.md

```diff
@@ -0,0 +1,169 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Raw RSA keyring example encoded wrapping keys
+
+Implementations of this example MUST follow the rules defined in
+[Example Templates](../../../examples.md#example-templates).
+
+This template shows loading from PEM encoding
+but implementations MAY demonstrate DER encoding instead
+if PEM support in that language is lacking.
+
+## Implementations
+
+- [Python (DEV)](https://github.com/aws/aws-encryption-sdk-python/blob/keyring/examples/src/keyring/raw_rsa/keypair_from_pem.py)
+- [Python Master Key Provider (DEV)](https://github.com/aws/aws-encryption-sdk-python/blob/keyring/examples/src/master_key_provider/raw_rsa/private_key_only_from_pem.py)
+- [Java (DEV)](https://github.com/aws/aws-encryption-sdk-java/blob/keyring/src/examples/java/com/amazonaws/crypto/examples/keyring/rawrsa/RawRsaDerEncoded.java)
```

Comment:
Should this URL be for DER?

I assume it's fine because "implementations MAY demonstrate DER encoding instead if PEM support in that language is lacking."

Reply:
Yeah, turns out it's messy to load from PEM in Java without manually unpacking it to DER or using BoucyCastle's helpers, so we went with DER for Java.